### DISCUSSION TITLE
Elements popping into place

### DISCUSSION THREAD

**Discussion by pesschap** [Nerwyn/material-you-theme#38] [Q&A] [answered]
Hi,

When I click on Overview from Settings, I notice elements popping into place, for example the views usually at the top disappear and move to the bottom. The elements on the dashboard shrink a little which is what was intended. This happens every time I switch between Overview and Settings. Is there a way to "preload" these changes so they don't appear every time I open Overview?

**Comment by Nerwyn** [comment, marked as answer]
This is the card-mod styles applying. There's nothing we can do about it here, other than avoiding styles within shadow-roots when possible. It would have to be improved in card-mod.
